### DISCUSSION TITLE
ควรแปล-ไม่แปลมากเท่าไหร่ดี

### DISCUSSION THREAD

**Discussion by srakrn** [srakrn/interpretable-ml-book-th#1] [Translation QAs]
ไม่แน่ใจว่าศัพท์เทคนิกนี่ควรแปลมากหรือน้อยเท่าไหร่ดีครับ

- คำบางคำ อย่างเช่น "model" ถ้าแปลเป็น "แบบจำลอง" จะเยิ่นเย้อไปหรือไม่
- ศัพท์ที่เป็นชื่อเฉพาะ เช่น "local surrogate" ถ้าไม่แปลก็ไม่เป็นไร แต่ถ้าแปลอาจจะได้เป็น "ตัวแทนสัมพัทธ์" ซึ่งยังฟังดูสมเหตุสมผลตามแบบเดียวกับที่เราแปลศัพท์คณิตศาสตร์
-  ศัพท์แปลกๆ ที่เราอาจจะได้ใช้กัน เช่น "convolution" ถ้าแปลจะได้เป็น "สังวัฒนาการ" ซึ่งชั่งน้ำหนักแล้วคนที่เคยได้ยินคำว่า convolution แต่ไม่เคยได้ยินคำว่าสังวัฒนาการ น่าจะเยอะกว่า แบบนี้ยังควรแปลหรือไม่

กำลังคิดว่ามี rule of thumb เช่น "แปลไปเลย แต่ถ้าแปลแล้วรู้สึกว่าคนอ่านต้องเปิดหาศัพท์อังกฤษ แปลว่าไม่ควรแปลแล้ว" น่าจะช่วยให้สบายใจในการแปลมากขึ้น--คิดว่าอย่างไรกันบ้างครับ

**Comment by 2yim** [comment]
ผมคิดว่าถ้าเราเลือกแปลคำศัพท์เฉพาะใดก็ตามเป็นภาษาไทย เมื่อเราไปเจอคำอื่นก็ต้องแปลเป็นไทยด้วย เพื่อให้มี consistency ตลอดทั้งเล่ม แต่เท่าที่ดู ศัพท์เฉพาะในทางนี้หลายคำ ไม่สามารถหาคำแปลที่เหมาะสมได้ อันนี้เป็นอคติส่วนตัว เพราะผมโตมากับอินเตอร์เฟซภาษาอังกฤษ คือ เมนูต่าง ๆ ใน Windows, เกม, หรือโปรแกรมต่าง ๆ เป็นภาษาอังกฤษ แต่เนื้อหาเป็นภาษาไทย จึงชอบอ่านคำเฉพาะที่ใช้อักษรโรมันไปเลย มันแยกชัดดี บางทีเห็รูปร่างของศัพท์ก็เดาคำได้เลย แต่ถ้าถอดเสียงเป็นอักษรไทย ถ้าไม่ใช่คำที่เจอบ่อยก็จะใช้เวลาประมวลผลนิดนึง ยิ่งคำแปล ถ้าแปลไม่สื่อความหมายมากพอก็จะยิ่งดีเลย์ ถ้าจะมีวงเล็บอักษรโรมัน ก็มีหลักว่าให้วงเล็บแค่ครั้งแรก วงเล็บบ่อย ๆ คนอ่านจะตาลายซะอีก 
   
    แต่อย่างไร อันนี้แล้วแต่กลุ่มเป้าหมายครับ ผมอาจจะเป็นคนแปลก : )

**Reply by srakrn** [reply]
ควรจะเป็นแบบนั้นครับ

เอาสามคำก่อนแล้วกันครับ 😂 "โมเดล" "แมชชีนเลิร์นนิง" "อัลกอริทึม" สามคำนี้ คิดว่าควรแปลไหมครับ

(ตอนนี้ผมใช้วิธีแปลไปก่อน เดี๋ยวจะสร้างตัว linting rules มาจับให้มัน auto replace น่าจะลดการทำมือไปมาก)
